### ISSUE TITLE
How to cite CF

### ISSUE THREAD

**Issue by lhmarsden** [cf-convention/cf-convention.github.io#559]
I recall that there is now a DOI for CF v1.11 but I haven't been able to find it on the website nor in the conventions document. There should perhaps be a "How to cite CF" button on the website (or similar) as well as something in the conventions document.

In the meantime, could someone please provide me with the recommended citation? I am making final touches to an article I would like to submit soon.

**Comment by sadielbartholomew** [comment]
I agree that this is far from transparent (though I appreciate the DOI is a work in progress and it is likely intended to be clarified much further after that is sorted) but is an important and common question anyone might wonder about.

For the above reasons I also think it would be good to add this ('How do I cite CF?') as a question under the FAQs page and then link that to the button or otherwise which provides the relevant citation or related information.

**Comment by lhmarsden** [comment]
Thanks, Jonathan, This is very help.

Luke
